src/ReportManager/DetailsSection/SchemaForm/SchemaFormContext/index.js

```diff
@@ -1,26 +1,3 @@
-import React, { createContext } from 'react';
-import PropTypes from 'prop-types';
+import { createContext } from 'react';
 
-export const SchemaFormContext = createContext(null);
```

Comment:
This file ended up being just a Context object to import everywhere and avoid circular dependencies. It only existed for the purpose of handling the `onFieldChange` method, but with the errors object, now it makes more sense to handle it in the parent `SchemaForm`. So, having the provider as an independent component don't have sense anymore.